site/chat/consumers.py

```diff
@@ -73,22 +82,67 @@ async def receive(self, text_data):
                 'user_id':self.user.id
                 # 'user': self.user.username+'('+str(self.user.id)+')'
             }
-        )
+        )   
+
+        #Esto debería hacerlo solo si esta habilitado el chat con bot
+        asyncio.create_task(self.send_to_bot(message))   
+        
+        
 
     # Receive message from room group
     async def chat_message(self, event):
         message = event['message']
         user = event['user']
         color = event['color']
         user_id = event['user_id']
-        own_msg = user_id == self.user.id
+        own_msg = user_id == self.user.id        
         # Send message to WebSocket
         await self.send(text_data=json.dumps({
             'message': message,
             'user': user,
             'color': color,
             'own_msg': own_msg
-        }))
+        }))       
+        
+        
+    async def send_to_bot(self,message):    
+        
+        client =httpx.AsyncClient()
+        #Responde el bot mockup
+        current_bot = await get_bot("repeat")
+        url= "http://127.0.0.1:8000/api/bots/"+ str(current_bot.id) +"/"       
+        data = {"prompt" : message}
+
+        #Prueba con ollama
+        # url = 'http://127.0.0.1:11434/api/generate'
+        # data = {    
+        # "model": "llama3",
+        # "prompt": message,    
+        # "stream": False
+        # }
+        response = await client.post(url,json=data)
+        
+        print(response)
+        print(response.json())   
+        
+        #Hay que revisar como guardar el chat,
+        #   Modificar models.chat para que pueda almacenar tanto bot como participante?
+        #   Crear un model de respuestas de bot que relacione mensaje de chat de participante con respuesta de bot?
+        #await store_chat(1, stage_name, message)
```

Comment:
Maybe we should consider that a bot is a type of participant, so we need to create a participant when we want to include a bot and just tag it as such